Server/docker-compose.yml

```diff
@@ -61,7 +61,7 @@ services:
       - 8055:8055
     volumes:
       - ./data/directus/uploads:/directus/uploads
-      - ./data/directus/extensions:/directus/extensions
+      - ../directus-extension-charging-stations-bundle:/directus/extensions/directus-extension-charging-stations-bundle
```

Comment:
that isnt what I am saying - the bundle name stays as is, but you put all your bundles and other extensions in one container folder so that you can do `directus-extensions/*:/directus/extensions`

Reply:
it is best to have `/directus-extensions` in root where you can then place /bundles/charging-stations-bundle whatever else... when running locally, these can all build to the same `/extensions` directory that is mounted that can with sourcemaps then support breakpoints. This way, you can load any number of extensions quickly and have hot reload as directus can pull new extensions if anything in that one folder changes...